cluster/manifests/api-monitoring-controller/deployment.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: "extensions/v1beta1"
+kind: Deployment
+metadata:
+  name: api-monitoring-controller
+  labels:
+    application: api-monitoring-controller
```

Comment:
The app doesn't exist, please create it.

Reply:
What do you mean? The app does exist in Kio: https://yourturn.stups.zalan.do/application/detail/api-monitoring-controller.

Reply:
Oops, sorry!